### PR TITLE
monit should be manually started (after-reboot)

### PR DESCRIPTION
**Pull request by javierwilson**
autostarting monit is crazy, since must services need to be started after "after-reboot" is ran (ecrypt mounted)

### CODE REVIEW

File: ansible/roles/monit/tasks/main.yml [hunk 1]
```diff
@@ -14,5 +14,7 @@
 
 - name: start monit
   become: yes
-  service: name="monit" state=started enabled=yes
-  tags: monit
+  service: name="monit" state=started enabled=no
```
Comment (snopoke) [inline ansible/roles/monit/tasks/main.yml]: why the change from `enabled=yes` to `enabled=no`?

Comment (javierwilson) [reply ansible/roles/monit/tasks/main.yml]: enabled means autostart, if we start monit as soon as ubuntu boots (with no other service started) monit will keep trying and failing to start services
